### PR TITLE
Get data to generate recommendation

### PR DESCRIPTION
**Pull request by msvinaykumar**
Earlier results data are retrieved outside performance profile util
now logic is moved into Perf profile.

### CODE REVIEW

**Comment by chandrams** [comment]
@msvinaykumar - Are these changes internal? Does these changes affect the API behavior or externals in any way?

File: src/main/java/com/autotune/analyzer/performanceProfiles/PerformanceProfileInterface/ResourceOptimizationOpenshiftImpl.java [hunk 1]
```diff
@@ -77,73 +80,92 @@ public List<KruizeRecommendationEngine> getEngines() {
...
+        /*
+            To restrict the number of rows in the result set, the Load results operation involves locating the appropriate method and configuring the desired limitation.
+            It's important to note that in order for the Limit rows feature to function correctly,
+            the CreateExperiment API must adhere strictly to the trail settings' measurement duration and should not allow arbitrary values
+        */
+        String experiment_name = kruizeObject.getExperimentName();
+        int limitRows = (int) ((
+                KruizeConstants.RecommendationEngineConstants.DurationBasedEngine.DurationAmount.LONG_TERM_DURATION_DAYS *
+                        KruizeConstants.DateFormats.MINUTES_FOR_DAY)
+                / kruizeObject.getTrial_settings().getMeasurement_durationMinutes_inDouble());
```
Comment (dinogun) [inline src/main/java/com/autotune/analyzer/performanceProfiles/PerformanceProfileInterface/ResourceOptimizationOpenshiftImpl.java:93]: Should this be `limitRows = LONG_TERM_MINS / durationMins + 1`?

Comment (dinogun) [reply src/main/java/com/autotune/analyzer/performanceProfiles/PerformanceProfileInterface/ResourceOptimizationOpenshiftImpl.java:93]: ok I see that it is existing code moved inside the `generateRecommendation` method now

**Review by dinogun — APPROVED** [review]
LGTM

**Pull request merged**